e2e/features/mock_exchange.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Using the mocked exchange backend
+  As a user I want to use the backend
+  To perform fake controlled requests
+
+  Background:
+    Given the network is running
```

Comment:
It seems this e2e test is just testing the mocked exchange. Do we really need a running network?
Also, do we really need to use e2e tests for this? I'd just use a standard go test instead.

Reply:
Agree. A pack of unit tests seems more suitable for this.

Reply:
Unless we plan to add e2e tests that really test how this exchange affect the network. Is it the plan @jgimeno ?

Reply:
We'll need e2e tests for the oracle client, which will use the mock exchange as data sources. So yes, there will eventually be full-network tests

Reply:
yep, this is just a test as an example on how to use it later for the oracle.

Reply:
But this e2e is just created so in the future it is easy to launch e2e tests using the mock server. What I can do is remove the feature files, or comment. @jmatosp @yourheropaul

Reply:
Maybe comment that line out for now so that the tests are quicker?